### ISSUE TITLE
Inserting Table generates incorrect HTML

### ISSUE THREAD

**Issue by aryadas98** [robert7/nixnote2#27]
## Expected Behavior
Inserting a table should produce the HTML:
```
<table style="..."><tbody>...</tbody></table>
```

## Actual Behavior
This HTML is produced:
```
<table style="..." <tbody=""><tbody>...</tbody></table>
```

This HTML is incorrect and produces errors while syncing.

## Steps to reproduce the problem:
1. Make a new note.
2. Insert a table

## Specifications
Version used: [v2.1.0-beta-3](https://github.com/robert7/nixnote2/releases/tag/v2.1.0-beta-3)

## Workaround
Manually edit the HTML.

**Comment by robert7** [comment]
hmm it seems, you are right. i'll fix it
i did not notice before, as normally the markup is fixed by "tidy" .. so you see, in my case I inserted 2 tables and the note was synced OK (and the markup is OK now)

![screen_20181003_03](https://user-images.githubusercontent.com/1135218/46425081-26a2c700-c73b-11e8-99a6-23c7def0d378.png)

**Comment by aryadas98** [comment]
The sync issue was happening on [baumgarr/nixnote2](https://github.com/baumgarr/nixnote2). So I thought it would happen here as well. Thanks for fixing it!

**Comment by robert7** [comment]
>The sync issue was happening on baumgarr/nixnote2...

...ah OK. The v2.1 has much improved the html cleanup (not perfect, but quite better). So chance of automatic cleanup is much higher here. But I'll fix the underlying issue of the incorrect html.

**Comment by robert7** [comment]
@aryadas98  this should be fixed in version 2.1.0-beta4h 
you can test it using the latest development AppImage https://github.com/robert7/nixnote2/releases/tag/continuous-develop
